### DISCUSSION TITLE
AI Assistant: are references file contents dynamically read?

### DISCUSSION THREAD

**Discussion by nrw303** [zed-industries/zed#17039] [Feature Requests]
This might be a silly question, but lets see :)

When using the AI Assistant, let's say I /file and reference some files, then during the course of the conversation these files are updated with new/changes code. conversation continues ... then let's say I want to reference those again; do I need to /file and reference them again since they've changed since they were last referenced? or does the assistant "keep track" of files that were referenced throughout the entire conversation?

**Comment by bajrangCoder** [comment]
I guess you need to refrence again after changes

**Reply by nrw303** [reply]
hi, thanks for replying, you guess, or you know ?

**Reply by bajrangCoder** [reply]
Now I know 😁(Previously I was not sure , but after testing I'm sure that it doesn't keep track of your file changes)

**Reply by nrw303** [reply]
thanks for clarifying!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
